Simulator/Vertices/AllVertices.h

```diff
@@ -33,6 +33,9 @@ using namespace std;
 // cereal
 #include "cereal/types/vector.hpp"
 
+// Utility function to convert a vertexType into a string.
+string vertexTypeToString(vertexType t);
+
```

Comment:
Should this be a member function of `Allvertices`? I'm not sure. Also, where did the `vertexType` definition go? This PR just shows it being deleted from `Global.h`.

Reply:
OK, I am clearly being thick. Is `VertexType.h` only being included in `RecordableBase.h`? Is that the best place?

Reply:
OK.